cider/src/flatten/structures/environment/program_counter.rs

```diff
@@ -76,59 +76,66 @@ impl ControlPoint {
     /// path is displayed in the minimal metadata path syntax.
     pub fn string_path(&self, ctx: &Context) -> String {
         let path = SearchPath::find_path_from_root(self.control_node_idx, ctx);
+        let control_map = &ctx.primary.control;
         let mut path_vec = path.path;
 
-        // Remove first element since we know it is a root
-        path_vec.remove(0);
         let mut string_path = String::new();
+
+        // Maybe parameterize prefix so we know which group we are on?
+        string_path.push('0');
         string_path.push('.');
-        let control_map = &ctx.primary.control;
-        let mut count = -1;
-        let mut body = false;
-        let mut if_branches: HashMap<ControlIdx, String> = HashMap::new();
+
+        // Remove first index
+        let node = path_vec.remove(0);
```

Comment:
This `.remove` is unnecessary and will force the vec to shift every element
over. In our case it would be better to make an iterator out of the vec, get the
next element for `node` and then consume the rest of the iterator in the for loop